src/Hng.Infrastructure/Hng.Infrastructure.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="8.0.6" />
```

Comment:
Remove all `Microsoft.AspNetCore.Identity.EntityFrameworkCore` reference as we are not using it.